downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/CreateReactome2BioSystems.java

```diff
@@ -0,0 +1,27 @@
+package org.reactome.release.downloadDirectory;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.biosystems.ReactomeToBioSystemsConverter;
+
+public class CreateReactome2BioSystems {
+	private static final Logger logger = LogManager.getLogger();
+	public static void execute(String host, String database, String username, String password, int port, String releaseNumber) throws IOException {
+		
+		logger.info("Running CreateReactome2BioSystems...");
+		// The last argument, 'BioSystems', specifies the output directory of the ReactomeToBioSystems.zip file. 
+		// The script removes all files within the directory, so only change the output directory to an empty or non-existent one
+		ReactomeToBioSystemsConverter.main(new String[] {host, database, username, password, Integer.toString(port), "BioSystems" });
+		String outpathName = releaseNumber + "/ReactomeToBioSystems.zip";
+		Files.move(Paths.get("BioSystems/ReactomeToBioSystems.zip"), Paths.get(outpathName), StandardCopyOption.REPLACE_EXISTING); 
+		FileUtils.deleteDirectory(new File("BioSystems"));
+		logger.info("Finished CreateReactome2BioSystems");
+	}
```

Comment:
Extracting constants might make configuring names and directories easier:

```
final String bioSystemsTempDir = "BioSystems";
final String bioSystemsFile = "ReactomeToBioSystems.zip";
final Path bioSystemsTempFilePath = Paths.get(bioSystemsTempDir, bioSystemsFile);
final Path bioSystemsTargetFilePath = Paths.get(releaseNumber, bioSystemsFile);


ReactomeToBioSystemsConverter.main(new String[] {host, database, username, password, Integer.toString(port), bioSystemsTempDir });
Files.move(bioSystemsTempFilePath, bioSystemsTargetFilePath, StandardCopyOption.REPLACE_EXISTING);
FileUtils.deleteDirectory(new File(bioSystemsTempDir));
```

Reply:
Maybe even make this configurable in a properties file if we decide to do that for the other steps

Reply:
I'm not sure I'd make `bioSystemsTempFilePath` and `bioSystemsTargetFilePath` as constants.

Reply:
Sure, they don't have to be constants.  I think keeping them as variables is clearer than inline calls to Paths.get() or concatenation, though.

Reply:
I used the suggestions above, with final Strings and Paths

Reply:
great!